src/scripts/clipperUI/components/previewViewer/previewComponentBase.tsx

```diff
@@ -70,8 +70,10 @@ export abstract class PreviewComponentBase<TState, TProps extends ClipperStatePr
 	}
 
 	private getPreviewSubtitle(): any {
-		let sourceUrlCitation = Localization.getLocalizedString("WebClipper.FromCitation")
-			.replace("{0}", this.props.clipperState.pageInfo ? this.props.clipperState.pageInfo.rawUrl : "");
+		let sourceUrlCitationPrefix = Localization.getLocalizedString("WebClipper.FromCitation")
+			.replace("{0}", ""); // TODO can we change this loc string to remove the {0}?
```

Comment:
Looks like in Javascript, if the 1st param of replace does not exist in the string, nothing happens. I think this change is fine, and we should have no problems changing the one stored on the server without having to worry about n-1 issues.